src/components/visualization/plotlyGraph/plotlyGraph.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import type {Meta, StoryFn} from '@storybook/react'
+import React from 'react'
+
+import {PlotlyGraph} from './plotlyGraph'
+import type {PlotlyGraphData} from "./plotlyGraph.types";
+
+const meta: Meta<React.ComponentProps<PlotlyGraphData>> = {
+    title: 'Rustic UI/Visualization/PlotlyGraph',
+    component: PlotlyGraph,
+    tags: ['autodocs'],
+    parameters: {
+        layout: 'centered',
+    },
+}
+
+export default meta
+
+meta.argTypes = {
+    ...meta.argTypes,
+    plotParams: {
+        table: {
+            type: {
+                summary: 'PlotlyJson\n',
```

Comment:
Should we add a description and point people to Plotly's doc: https://github.com/plotly/react-plotly.js/?tab=readme-ov-file#basic-props